src/components/NodeToolbar/AccessibilityEditor/saveStatus.js

```diff
@@ -0,0 +1,63 @@
+import { t } from 'c-3po';
+import fetch from '../../../lib/fetch';
+import config from '../../../lib/config';
+import { wheelmapFeatureCache } from '../../../lib/cache/WheelmapFeatureCache';
+import { wheelmapLightweightFeatureCache } from '../../../lib/cache/WheelmapLightweightFeatureCache';
+import { YesNoLimitedUnknown } from '../../../lib/Feature';
+
+type ExternalSaveOptions<T> = {
+  featureId: string,
+  value: T,
+  onSave: ?((T) => void),
+  onClose: (() => void),
+};
+
+type SaveOptions<T> = ExternalSaveOptions<T> & {
+  url: string,
+  propertyName: string,
+  jsonPropertyName: string,
+};
+
+
+function save<T>(options: SaveOptions<T>): Promise<Response> {
+  const { url, value, featureId, propertyName } = options;
+
+  const formData = new FormData();
+  formData.append(options.jsonPropertyName, value);
+
+  const requestOptions = {
+    method: 'PUT',
+    body: window.cordova ? { toilet: value } : formData,
+    cordova: true,
+    serializer: 'urlencoded',
+  };
+
+  [wheelmapFeatureCache, wheelmapLightweightFeatureCache].forEach(cache => {
+    if (cache.getCachedFeature(String(featureId))) {
+      cache.updateFeatureAttribute(String(featureId), { [propertyName]: value });
+    }
+  });
+
+  return fetch(url, requestOptions)
+    .then(() => {
+      setTimeout(() => {
+        if (typeof options.onClose === 'function') options.onClose();
+        if (typeof options.onSave === 'function') options.onSave(value);
+      }, 50);
+    })
+    .catch((e) => {
+      window.alert(t`Sorry, could not mark this place because of an error: ${e}`);
+    });
+}
+
+
+export function saveToiletStatus(options: ExternalSaveOptions<YesNoUnknown>) {
+  const url = `${config.wheelmapApiBaseUrl}/nodes/${options.featureId}/update_toilet.js`;
+  return save({ ...options, url, propertyName: 'wheelchair_toilet', jsonPropertyName: 'toilet' });
```

Comment:
So this can also save toilet=no now?

Reply:
Yes, it can – Did I miss something? Not sure what you mean with this…